Tetfuza/TetfuzaCLI/SystemConsoleDisplay.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tetfuza;
+using Tetfuza.Interfaces;
+
+namespace TetfuzaCLI
+{
+    public class SystemConsoleDisplay : IDisplay
+    {
+        public void GameScreen(TetfuzaBackend gameState)
```

Comment:
I feel like you might want to use StringBuilder to make a screen rep in memory first and then blit it to the screen in one big console.write